stdlib/public/Concurrency/TaskGroup.cpp

```diff
@@ -593,8 +593,10 @@ struct TaskGroupStatus {
 #elif defined(STDERR_FILENO)
     write(STDERR_FILENO, message, strlen(message));
 #endif
-#if defined(SWIFT_STDLIB_HAS_ASL)
-    asl_log(nullptr, nullptr, ASL_LEVEL_ERR, "%s", message);
+#if defined(SWIFT_STDLIB_HAS_OS_LOG)
+  os_log_t type = os_log_create("SwiftRuntime", "Error");
```

Comment:
We shouldn't create a new log object every time. For this routine, I bet we can get away with `[OS_LOG_DEFAULT](https://developer.apple.com/documentation/os/os_log_default)`

Reply:
As an aside, the old ASL implementation appears to violate the thread safety constraints of the API 

> When logging is done from a single thread, a NULL value may be used in
     any of the routines that require an aslclient argument.  In this case,
     the library will open an internal client handle on behalf of the applica-tion. application.
     tion.

>     If multiple threads must do logging, or if client options are desired,
     then the application should call asl_open to create a client handle for
     each thread.

whoops